portal-api/src/v1/controllers/utilisation-report-service/utilisation-report-upload.controller.js

```diff
@@ -62,35 +64,74 @@ const sendEmailToBankPaymentOfficerTeam = async (reportPeriod, bankId, submitted
   }
 };
 
+/**
+ * Saves file to Azure in utilisation-reports ShareClient, returns the file storage info
+ * @param {object} file
+ * @param {string} bankId - bank id as a string
+ * @returns {Promise<{fileInfo: object, error: boolean}>} - if file is not saved error is true, otherwise returning
+ * azure storage details with folder & file name, full path & url.
+ */
+const saveFileToAzure = async (file, bankId) => {
+  try {
+    console.info(`Attempting to save utilisation report for bank: ${bankId}`);
+    const { originalname, buffer } = file;
+
+    const fileInfo = await uploadFile({
+      fileshare: FILESHARES.UTILISATION_REPORTS,
+      folder: bankId,
+      filename: formatFilenameForSharepoint(originalname),
+      buffer,
+      allowOverwrite: true,
+    });
+
+    console.info(`Successfully saved utilisation report for bank: ${bankId}`);
+    return { fileInfo, error: false };
+  } catch (error) {
+    console.error('Failed to save utilisation report: %O', error);
+    return { fileInfo: null, error: true };
+  }
+};
+
 const uploadReportAndSendNotification = async (req, res) => {
   try {
+    const { file } = req;
+
     const { reportPeriod, reportData, month, year, user } = req.body;
     const parsedReportData = JSON.parse(reportData);
     const parsedUser = JSON.parse(user);
 
-    // TODO: FN-967 save file to azure
-    // const file = req.file;
-
-    // if (!file) return res.status(400).send();
+    if (!file) return res.status(400).send();
```

Comment:
nit: As a rule of thumb I almost never use an `if` block without using `{}` to scope the change. This helps prevent unexpected errors as the `{}` make absolutely clear what is being executed when the `if` predicate is true.
```
if (!file) {
  return res.status(400).send();
}
```